controllers/component_build_controller.go

```diff
@@ -45,6 +45,7 @@ import (
 const (
 	BuildRequestAnnotationName                    = "build.appstudio.openshift.io/request"
 	BuildRequestTriggerSimpleBuildAnnotationValue = "trigger-simple-build"
+	BuildRequestTriggerPaCBuildAnnotationValue    = "trigger-pac-build"
```

Comment:
do you think it would be good to cover e2e-tests for this annotation support [here](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/build/build.go#L717)?

Reply:
I know nothing about our e2e-tests, but yeah there should be test for this there

Reply:
created a jira ticker to cover that